lib/rules/jsx-curly-brace-presence.js

```diff
@@ -116,15 +116,15 @@ module.exports = {
     // --------------------------------------------------------------------------
 
     return {
-      JSXExpressionContainer: function(node) {
+      JSXExpressionContainer: node => {
```

Comment:
I made the change to maintain consistencies across files, but I guess it doesn't matter anyway. I have changed it back to using the arrow function.